ansible/group_vars/all.yml

```diff
@@ -40,6 +40,9 @@ insecure_registrys:
 # The port that the Kubernetes apiserver component listens on.
 kube_master_api_port: 443
 
+# Endpoint for Kubernetes apiservers
+kube_apiserver_endpoint: "{% if groups['masters']|length > 1 %}{{ kube_apiserver_vip }}{% else %}{{ groups['masters'][0] }}{% endif %}"
```

Comment:
Instead of defining kube_apiserver_endpoint, I would expect to see kube_apiserver_vip instead. kube_apiserver_vip would be commented-out by default. Inline comment would include basic instructions on how to use the var, ie for ha, when defining x number of masters in inventory, etc..